man/avahi-daemon.conf.5.xml.in

```diff
@@ -201,10 +201,13 @@
       ("yes" or "no"). Enable wide-area DNS-SD, aka
       DNS-SD over unicast DNS. If this is enabled only domains
       ending in .local will be resolved on mDNS, all other domains
-      are resolved via unicast DNS. If you want to maintain multiple
-      different multicast DNS domains even with this option enabled
-      we encourage you to use subdomains of .local, such as
-      "kitchen.local". This option defaults to "no".</p>
+      are resolved via unicast DNS. When this is enabled, unless
+      explicitly specified reverse lookup will go over unicast DNS
+      and fall back to mDNS if unicast DNS lookup fails.
+      If you want to maintain multiple different multicast DNS
+      domains even with this option enabled we encourage you to
+      use subdomains of .local, such as "kitchen.local".
```

Comment:
I am not sure subdomains of ``.local`` are well supported for mdns. I think at least nss-mdns expect only 2 label queries like something.local. I have never tested it with subdomains and I am not sure they work as expected. Has it been successfully tested before recommending them?

Reply:
> Has it been successfully tested before recommending them?
I don't know, this change only introduces the impact to reverse lookups, it doesn't touch that subdomain part:)

Reply:
> I am not sure subdomains of .local are well supported for mdns

avahi supports all sorts of domains. If `domain-name` is set to, say, `wat.local` other machines can see that:
```
Timestamp     A/R    Flags if Hostname                               Address                                      TTL
17:19:59.247  Add 40000003 13 H.wat.local.                           FE80:0000:0000:0000:7299:E626:F044:3DB9%bridge100 120
17:19:59.247  Add 40000002 13 H.wat.local.                           192.168.56.4                                 120
```
It even works without `.local`: https://github.com/avahi/avahi/issues/155. I'm not saying it should be recommended but it certainly should be documented. Either way it isn't related to this PR.

Reply:
I replaced the last "lookup" with "lookups" and squashed all the commits. I'll merge the PR once the CI is green. Thanks!